app/src/main/java/dev/andrylat/raqimbek/bankingutils/CardValidator.java

```diff
@@ -0,0 +1,124 @@
+package dev.andrylat.raqimbek.bankingutils;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.ArrayList;
+import java.util.stream.IntStream;
+
+public class CardValidator {
+    public CardValidationInfo checkCardNumber(String input) {
+        var errors = new ArrayList<String>();
+        var isValid = false;
```

Comment:
Why do you create this variable here and don't use during next 10 lines? You should define variable before usage. In other case when I read the code, I need to remember variables which are not used.